packages/web-react/src/components/Field/README.md

```diff
@@ -0,0 +1,21 @@
+# ValidationText
+
+The validationText subcomponent displays validation messages for Field components like TextField, TextArea, FileUploader, tsc.
+
+```jsx
+import { ValidationText } from '@lmc-eu/spirit-web-react/components';
+```
+
+```jsx
+<ValidationText elementType="div" className="TextField__message" validationText="This field is required" />
+```
+
+## ValidationText
+
+**Available props**
+
+| Name             | Type                 | Default | Required | Description                          |
+| ---------------- | -------------------- | ------- | -------- | ------------------------------------ |
+| `elementType`    | `span`, `div`        | `div`   | no       | Type of element used as main wrapper |
```

Comment:
it could be any element, couldn't it?

Reply:
also it is only applied when the text isn't an array

Reply:
> it could be any element, couldn't it?

Technically yes. But I prefer to suggest only elements that work well with this.